conans/client/profile_loader.py

```diff
@@ -38,7 +38,10 @@ def __init__(self, text):
                 include = include[:-1]
                 self.includes.append(include)
             else:
-                name, value = line.split("=", 1)
+                try:
+                    name, value = line.split("=", 1)
+                except ValueError as error:
+                    raise ConanParsingError("Error while parsing line %i: '%s'" % (counter, line))
```

Comment:
Let's throw a generic ``ConanException``. New exceptions will be introduced from now on, only if it is going to be captured and processed somewhere else. Let's keep it simple.